WooCommerce/Classes/ViewRelated/Products/Edit Product/Inventory Settings/ProductInventoryEditableData.swift

```diff
@@ -15,7 +15,7 @@ extension ProductInventoryEditableData {
     init(productModel: ProductFormDataModel) {
         self.sku = productModel.sku
         self.manageStock = productModel.manageStock
-        self.soldIndividually = (productModel as? Product)?.soldIndividually
+        self.soldIndividually = (productModel as? EditableProductModel)?.product.soldIndividually
```

Comment:
The `soldIndividually` could probably be hidden behind the `ProductFormDataModel` too, right?

Reply:
yes great catch! it can be in `ProductFormDataModel` now without making `ProductVariation` support a field that doesn't apply to it 😄 updated here: https://github.com/woocommerce/woocommerce-ios/pull/2597/commits/fd7d9cf161c6d2ce7db6aefdd86594b872bdc11e